testing/run-scripts/benchmark_transformers.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/python3
+
+import argparse
+import csv
+import io
+import subprocess
+import sys
+import time
+import urllib.parse
+
+FLOOD_SIZE_MB = 64
+FLOOD_MAX_SPEED = '5M'
```

Comment:
Perhaps standardize this a bit?  '64M' to include the unit suffix, or 5 to exclude it (putting the +'M' in the call to subprocess.check_output().